ophyd/areadetector/detectors.py

```diff
@@ -109,6 +110,11 @@ class BrukerDetector(DetectorBase):
     cam = C(cam.Andor3DetectorCam, 'cam1:')
 
 
+class DexelaDetector(DetectorBase):
+    _html_docs = ['DexelaDoc.html']
+    cam = C(cam.DexelaDetectorCam, 'cam1:')
```

Comment:
Do we also need to add `DexelaDetectorCam` into `cam` in order to get this to work? I just want to make sure that if it is added here it will work as expected.